packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -310,7 +314,11 @@ test.describe('severity-2 #smoke', () => {
       page,
       target,
       pages: { signupReact, relier, subscribe, login },
-    }) => {
+    }, { project }) => {
+      test.skip(
+        project.name === 'production',
+        'no test products available in prod'
+      );
```

Comment:
I DM'd with Reino and we came to the conclusion that for this PR we should skip this test in production since we don't currently have any other SubPlat smoke tests in prod and there is a question about metrics impact. [I left a comment here](https://mozilla-hub.atlassian.net/browse/FXA-7861?focusedCommentId=814053).